src/main/java/com/amazon/opendistroforelasticsearch/sql/query/maker/Maker.java

```diff
@@ -319,6 +320,7 @@ private ToXContent make(Condition cond, String name, Object value) throws SqlPar
                 toXContent = QueryBuilders.idsQuery(type).addIds(ids);
                 break;
             case NESTED_COMPLEX:
+            case NE_NESTED_COMPLEX:
                 if (value == null || !(value instanceof Where)) {
                     throw new SqlParseException("unsupported nested condition");
```

Comment:
We have another exception class `SqlFeatureNotImplementedException`. Not sure which one is better fit here.

Reply:
Since it is existing exception handling procedure in the Maker, keep it for now.